### PR TITLE
.NET Core 3.1.1 Intellisense (English)

### PR DESCRIPTION
**Pull request by carlossanlop**
Bump for 3.1.1

### CODE REVIEW

File: standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   </assembly>
   <members>
     <member name="T:System.Buffers.ArrayPool`1">
-      <summary>Provides a resource pool that enables reusing instances of type <see cref="T[]" />.</summary>
```
Comment (safern) [inline standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml]: Are this changes correct? Should we remove the references to `T[]` and/or `T`?

File: standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <member name="M:System.Buffers.ArrayPool`1.Rent(System.Int32)">
       <summary>Retrieves a buffer that is at least the requested length.</summary>
       <param name="minimumLength">The minimum length of the array.</param>
-      <returns>An array of type <see cref="T[]" /> that is at least <paramref name="minimumLength" /> in length.</returns>
+      <returns>An array of type T that is at least <paramref name="minimumLength" /> in length.</returns>
```
Comment (safern) [inline standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml]: Ditto.

File: standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml (hunk 1 above)
Comment (carlossanlop) [reply standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml]: All these files were built by the Docs repo. Let me find the xml that generated it.

Comment (carlossanlop) [reply standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml]: Yes, that's exactly how we merged it and how it was generated.
https://github.com/dotnet/dotnet-api-docs/blob/6f84948d2563a9b53a9cfb292d78a2caecacc2dd/xml/System.Buffers/ArrayPool%601.xml#L1

We could improve it, but the change will go into a future .NET Core IntelliSense version.

File: standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml (hunk 2 above)
Comment (carlossanlop) [reply standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml]: See answer. That's how we have it.
https://github.com/dotnet/dotnet-api-docs/blob/6f84948d2563a9b53a9cfb292d78a2caecacc2dd/xml/System.Buffers/ArrayPool%601.xml#L1

File: standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml (hunk 1 above)
Comment (carlossanlop) [reply standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Buffers.xml]: The cref link was removed on purpose in this PR:
https://github.com/dotnet/dotnet-api-docs/pull/3466

**Comment by ahsonkhan** [comment]
Looks great. Thanks!

File: standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Runtime.Numerics.xml [hunk 3]
```diff
@@ -148,21 +151,7 @@
...
-  The current instance is less than <paramref name="other" />.  
-  
-  Zero  
-  
-  The current instance equals <paramref name="other" />.  
-  
-  Greater than zero  
-  
-  The current instance is greater than <paramref name="other" />.</returns>
+Return valueDescriptionLess than zeroThe current instance is less than <paramref name="other" />.ZeroThe current instance equals <paramref name="other" />.Greater than zeroThe current instance is greater than <paramref name="other" />.</returns>
```
Comment (ahsonkhan) [inline standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Runtime.Numerics.xml]: @mairaw - is this expected as the intellisense output for tables? I would imagine this looking odd in VS. It makes sense (and looks great) on the docs site.

https://github.com/dotnet/dotnet-api-docs/blob/6d40805a3453d59a5707781cbae4553c82dc2321/xml/System.Numerics/BigInteger.xml#L1133-L1152

https://docs.microsoft.com/en-us/dotnet/api/system.numerics.biginteger.compareto?view=netcore-3.0#System_Numerics_BigInteger_CompareTo_System_UInt64

image

Comment (mairaw) [reply standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Runtime.Numerics.xml]: It probably doesn't look great. Tables have always been problematic for IntelliSense and it's been in our backlog to fix them (https://github.com/dotnet/dotnet-api-docs/issues/2264).
But the new IntelliSense files that I'm currently testing output something better:

image

**Pull request merged**